### PR TITLE
mda: extend MDAnalysis plugin by bond information.

### PR DESCRIPTION
**Pull request by KaiSzuttor**

### CODE REVIEW

**Comment by KaiSzuttor** [comment]
there is an issue for bonds with more than two partners to be resolved

File: src/python/espressomd/MDA_ESP/__init__.py [hunk 1]
```diff
@@ -164,6 +178,7 @@ def parse(self):
                  Tempfactors(np.zeros(natoms)),
                  ICodes(np.array([' '], dtype=object)),
                  Charges(np.array(charges)),
+                 Bonds(bonds)
```
Comment (fweik) [inline src/python/espressomd/MDA_ESP/__init__.py]: Should the angles and dihedrals be here too?

Comment (jngrad) [reply src/python/espressomd/MDA_ESP/__init__.py]: yes, it's already fixed in the branch, but GitHub is not updating it (see 03a07713ae)

**Review by fweik — APPROVED** [review]
Great that there is a test now.

**Review by jngrad — APPROVED** [review]

**Pull request merged**